hs_core/views/__init__.py

```diff
@@ -863,7 +863,7 @@ def file_download_url_mapper(request, shortkey):
         zipped = True
     aggregation_name = None
     if aggregation:
-        aggregation_name = public_file_path.split("/")[-1]
+        aggregation_name = filepath
         zipped = True
```

Comment:
an aggregation download has to be zipped...interesting.
I guess we should ideally remove the "Download" option from the UI and only have the "Download Zipped"
![image](https://github.com/user-attachments/assets/27e4ff65-b0c9-4f21-bdad-7d24cc6e0a1e)
But that is likely for another day...